docs/features.md

```diff
@@ -52,6 +52,16 @@ The `Disruption` resource uses [label selectors](https://kubernetes.io/docs/conc
 
 **NOTE:** If you are targeting pods, the disruption must be created in the same namespace as the targeted pods.
 
+### DynamicTargeting
+
+By default, targeting is limited to a single target selection at the disruption's creation. It allows for more controlled disruption impact and propagation, as the targets will never change and canbe compensated in case of issue. Its major limit is not being able to follow targets through deployments/rollouts.
```

Comment:
I meant if disruption neutralises the pods, new pods can be deployed to keep the service active/bring back service activity in case of an issue with the disruption.
I agree it's not a clean formulation.